src/Core/MailTemplates/Handlebars/InitiateDeleteOrganzation.text.hbs

```diff
@@ -0,0 +1,17 @@
+{{#>BasicTextLayout}}
+We recently received your request to permanently delete the following Bitwarden organization:
+
+- Name: {{OrganizationName}}
+- ID: {{OrganizationId}}
+- Created: {{OrganizationCreationDate}} at {{OrganizationCreationTime}} {{TimeZone}}
+- Plan: {{OrganizationPlan}}
+- Number of seats: {{OrganizationSeats}}
+- Billing email address: {{OrganizationBillingEmail}}
+
+Click the link below to complete the deletion of your organization.
+
+If you did not request this email to delete your Bitwarden organization, you can safely ignore it.
```

Comment:
Maybe a nit, but if you didn't request this email you should probably be worried! Someone is trying to delete your organization. Maybe "If you did not request this email to delete your Bitwarden organization, please contact us." ?

Reply:
Good point. I am not opposed to changing that.